### ISSUE TITLE
Only use relative font sizes

### ISSUE THREAD

**Issue by zner0L** [datenanfragen/website#905]
To allow users to enlarge the website to a size they can read better, relative font sizes like `rem` should be used in the CSS. We are very inconsistent about this at the moment and should convert all pixel font sizes to visually equivalent relative ones.

Changing the font sizes should also not break the website. We should test text enlarging and how some components react to it. Also, for the mobile and electron App, were user styles might not be available, we should provide an accessibility setting.
